build.sh

```
@@ -75,16 +75,37 @@ print_status "Copying demo files..."
+        print_status "Running wasm-opt on $(basename "$path")..."
+        wasm-opt -Oz --strip-dwarf --strip-producers \
+            -o "${path}.opt" "$path"
+        mv "${path}.opt" "$path"
+    fi
+}
+
+# Print WASM sizes (uncompressed and gzipped)
+print_wasm_size() {
+    local path="$1"
+    if [ -f "$path" ]; then
+        local size=$(ls -lh "$path" | awk '{print $5}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)